src/browser_app_skeleton/src/components/shared/layout_head.cr

```diff
@@ -12,7 +12,7 @@ class Shared::LayoutHead < BaseComponent
 
       # Development helper used with the `lucky watch` command.
       # Reloads the browser when files are updated.
-      live_reload_connect_tag if LuckyEnv.development?
+      live_reload_connect_tag
```

Comment:
Oh, we already had that? I totally forgot. I should have looked here first. So this goes back to https://github.com/luckyframework/lucky_cli/issues/782 I vaguely remember there being a few that was confused about this. Though the conditional is actually irrelevant, it seems it provides sort of "documentation" beyond the comment itself.

Here's the PR that added it in https://github.com/luckyframework/lucky_cli/pull/794

I'm not sure which is a better default :man_shrugging:

Reply:
The fact that the Luckyframework website is different confused me.

Maybe we should add a comment on the same line?
```crystal
live_reload_connect_tag # automatically omitted for production
```

Reply:
The website is probably different because that change was missed after the last release. I guess a comment could work too